src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -101,7 +101,10 @@ private static Person createEditedPerson(Person personToEdit, EditPersonDescript
         Address updatedAddress = editPersonDescriptor.getAddress().orElse(personToEdit.getAddress());
         Set<Tag> updatedTags = editPersonDescriptor.getTags().orElse(personToEdit.getTags());
 
-        return new Person(updatedName, updatedPhone, updatedEmail, updatedAddress, updatedTags);
+        // Use the same person for existing fields, but copies the object for every thing else
+        // TODO: change this when the command is updated
+        return new Person(updatedName, updatedPhone, updatedEmail, personToEdit.getIdentityCardNumber(),
```

Comment:
should there still be the email? because we dont have the email field and the appending of the email field is also removed from the Messages.java above

Reply:
@tengcharmaine fixed Messages.java to include the email field (and all the new fields as well).